src/index.ts

```diff
@@ -1,5 +1,76 @@
-export class TestClass {
-    constructor() {
+import { Guid } from 'guid-typescript';
+import { AzureWebConfig } from './azure-web-config';
+import { SignalRService } from './signalr-service';
+import { ServiceBusService } from './service-bus-service';
+import { Observable, of } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+export class AzureWebEvent {
+    private sbClient: ServiceBusService;
+    private srClient: SignalRService;
+    public initialized: boolean = false;
+
+    /**
+     * Create a new instance of AzureWebEvent.
+     */
+    constructor() { }
+
+    /**
+     * Initialize the communication with azure.
+     * @param nameSpace Azure namespace.
```

Comment:
This is not really a nameSpace but rather a part of the endpoint. I think it will be clearer to pass the endpoint in here as a whole. And not building it using the `https://${endpoint}-functions.azurewebsites.net/api` template.

Reply:
changed